config-connector/solutions/iam/helm/kms-key-ring/templates/kmskeyring.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: kms.cnrm.cloud.google.com/v1beta1
+kind: KMSKeyRing
+metadata:
+  name: {{ .Values.KMSKeyRing.name }}
```

Comment:
name and location are also required, can you please add?

Reply:
@AlexBulankou In our previous helm solutions, only the fields that have PLACEHOLDER values in the corresponding kpt solutions are marked as required. Do we want to change that strategy?

Reply:
Let's keep the same approach as before then.